blivet/formats/luks.py

```diff
@@ -303,11 +303,12 @@ def _create(self, **kwargs):
             if luks_data.pbkdf_args:
                 self.pbkdf_args = luks_data.pbkdf_args
             else:
-                mem_limit = crypto.calculate_luks2_max_memory()
-                if mem_limit:
-                    self.pbkdf_args = LUKS2PBKDFArgs(max_memory_kb=int(mem_limit.convert_to(KiB)))
-                    luks_data.pbkdf_args = self.pbkdf_args
-                    log.info("PBKDF arguments for LUKS2 not specified, using defaults with memory limit %s", mem_limit)
+                if not crypto.is_fips_enabled():
```

Comment:
Would it be possible to add a comment here to make it obvious why the code is not needed in FIPS mode?

Reply:
Ok, I added a comment.